10-JavaScript/exercises/functions-worksheet.md

```diff
@@ -79,4 +97,7 @@ console.log(blabbermouth.name);
 ```
 
 5.
-Create a constructor function called `Square()` to create square objects with a side property and a method to calculate the area. Use the `Square()` function to create a square object.
+Create a class with constructor function called `Rectangle` to create rectangle objects with length and width properties and a method to calculate the area. Use the `Rectangle` class to create a rectangle object.
```

Comment:
I think the order of vocab is confusing: "Create a class called `Rectangle` with a constructor function. This class should create...."